### PR TITLE
Feature/s7 auto script

### LINKED ISSUE #356: Folder to automatically run scripts
Enable a folder, as a volume on docker, that allows for executing sql scripts for example on an automatically way.
This could also be done with python scripts.

### PR DESCRIPTION
**Pull request by JaimeSotomayor**
Closes #356

### CODE REVIEW

File: cornflow-dags/DAG/auto_scripts/automatic_scripts.py [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+
+                with open("output.txt", "r") as f:
+                    log = f.read()
+
+                os.remove("output.txt")
+
+                if error != 0:
+                    print(f"Something went wrong: {str(log)}")
+
+                error = 1
```
Comment (marioncottard) [inline cornflow-dags/DAG/auto_scripts/automatic_scripts.py:83]: I think you forgot to remove that last line, `error = 1`

**Review by marioncottard — COMMENTED** [review]
Other than that, it looks good to me

File: cornflow-dags/DAG/auto_scripts/automatic_scripts.py (hunk 1 above)
Comment (ggsdc) [reply cornflow-dags/DAG/auto_scripts/automatic_scripts.py:83]: @JaimeSotomayor fix this

File: cornflow-dags/DAG/auto_scripts/automatic_scripts.py [hunk 2]
```diff
@@ -59,14 +57,18 @@ def move_script(auto_script, log="", error=0):
     print(f"Script {auto_script} moved out of folder")
 
 
-def execute_scripts():
+def execute_scripts(
+    script_folder="/usr/local/airflow/dags/auto_scripts/scripts_to_execute",
+    destination_folder="/usr/local/airflow/dags/auto_scripts/executed_scripts",
+):
+    print(os.getcwd())
```
Comment (marioncottard) [inline cornflow-dags/DAG/auto_scripts/automatic_scripts.py:64]: Is this print necessary ?

File: cornflow-dags/tests/test_dags_af.py [hunk 3]
```diff
@@ -66,3 +65,12 @@ def test_access_all_variables(self):
             variable["key"] for variable in client.get_all_variables()["variables"]
         ]
         self.assertEqual(len(variables), len(existing_apps) * 2)
+
+    def test_auto_scripts(self):
+        print(os.getcwd())
```
Comment (marioncottard) [inline cornflow-dags/tests/test_dags_af.py:70]: Same here, is this necessary and if not, can you remove it ?

**Review by marioncottard — APPROVED** [review]

**Review by ggsdc — APPROVED** [review]

**Pull request merged**